splitgraph/commandline/cloud.py

```diff
@@ -1213,6 +1218,50 @@ def seed_c(remote, seed, github_repository, directory):
     click.echo(f"Splitgraph project generated in {os.path.abspath(directory)}.")
 
 
+@click.command("tunnel")
+@click.option("--remote", default="data.splitgraph.com", help="Name of the remote registry to use.")
+@click.option(
+    "--repositories-file", "-f", default=["splitgraph.yml"], type=click.Path(), multiple=True
+)
+@click.argument("repository", type=str)
+def tunnel_c(remote, repositories_file, repository):
+    """
+    Start the tunnel client to make tunneled external repo available.
+
+    This will load a splitgraph.yml file and tunnel the host:port address of the
+    external repository specified in the argument.
+    """
+
+    from splitgraph.cloud.project.utils import load_project
+
+    repo_yaml = load_project(repositories_file)
+    tunneled_repo = None
+    for r in repo_yaml.repositories:
+        if f"{r.namespace}/{r.repository}" == repository:
+            tunneled_repo = r
+
+    if tunneled_repo is None:
+        raise click.UsageError(
+            "Repository %s not found in %s" % (repository, ", ".join(repositories_file))
+        )
+
+    config_dir = os.path.dirname(get_singleton(CONFIG, "SG_CONFIG_FILE"))
+    # TODO: Get current version of rathole client for architecture.
+    # user must manually download rathole for now
+    rathole_client_binary_path = os.path.join(config_dir, "rathole")
+
+    from splitgraph.cloud import GQLAPIClient
+
+    client = GQLAPIClient(remote)
+    provisioning_token = client.get_tunnel_provisioning_token(
+        tunneled_repo.namespace, tunneled_repo.repository
+    )
+    rathole_client_config_path = write_rathole_client_config(
+        provisioning_token, tunneled_repo, config_dir
+    )
+    launch_rathole_client(rathole_client_binary_path, rathole_client_config_path)
```

Comment:
The goal is to have `sgr` download the appropriate `rathole` build for the platform (or throw an error if it can't find it on the github releases page for rathole).

Reply:
Ah cool -- I thought expecting `rathole` in `~/.splitgraph` was going to stay the default.